lib/dataSources.js

```diff
@@ -38,12 +38,13 @@ const add = (conn, name, options) => {
   }).then(httpBody);
 };
 
-const update = (conn, name, options) => {
+const update = (conn, name, options, params) => {
```

Comment:
I think we usually equate `params` to be `queryParams` (`ex.com?query=query`), so this may be better named as something like `opts`. Otherwise, I do lean towards not append to the body unless a parameter is explicitly defined, but don't feel too strongly about that since I think we do that elsewhere but I can't remember where.

Reply:
Hmm, I like `requestOptions` (currently only used in `virtualGraphs.mappings`)

Reply:
ooh yep, that's a good way to differentiate it from the actual object options, nice